src/transformers/models/roberta/modeling_roberta.py

```diff
@@ -676,8 +756,14 @@ def forward(
 
         device = input_ids.device if input_ids is not None else inputs_embeds.device
 
+        # past_key_values_length
+        past_key_values_length = past_key_values[0][0].shape[2] if past_key_values is not None else 0
+
+        batch_size, seq_length = input_shape
+        real_seq_length = seq_length + past_key_values_length
```

Comment:
(nit) move `seq_length + past_key_values_length` directly to below to save 1 line of code